jenkins/toggle-webhook-processor.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+import sys
+import logging
+
+
+def main():
+    try:
+        # connect to aws lambda
+        client = boto3.client('lambda')
+    except Exception, e:
+        # if connection failed, log error
+        logging.error(e)
+        sys.exit(1)
+
+    # ensure the number of arguments is correct
+    num_args = len(sys.argv)
+
+    if num_args == 2:
+        # parse toggle argument
+        arg = sys.argv[1]
+
+        try:
+            # find event source mapping for webhook_processor
+            response = client.list_event_source_mappings(
+                FunctionName='webhook_processor'
+            )
+            uuid = response["EventSourceMappings"][0]["UUID"]
+        except:
+            # if none found, lambda does not exist
+            logging.error(
+                "No EventSourceMappings found for that lambda function name"
+            )
+            sys.exit(1)
+
+        # call method according to the passed argument
+        if arg == "ON":
+            toggle(client, uuid, True)
+        elif arg == "OFF":
+            toggle(client, uuid, False)
+        else:
+            logging.error("Invalid argument. Please specify ON or OFF")
+    else:
+        logging.error("Incorrect number of arguments")
+        sys.exit(1)
+
+
+def toggle(client, uuid, enabled):
+    # find the previous state, update, and show new state
+    previous_state = get_current_state(client, uuid)
+    client.update_event_source_mapping(UUID=uuid, Enabled=enabled)
+    current_state = get_current_state(client, uuid)
+
+    print "The state was: " + previous_state
+    print "The state is now: " + current_state
```

Comment:
3 nits:
* try to standardize on a logging mechanism, as opposed to split between print statements and logging. You also may want to initialize a logger (but again, this is a super-nit)
* When you say "incorrect number or arguments" could you describe the required argument?
* Personal taste, but it would be easier to debug if you interleave the log statements and the code.